### PR TITLE
sql quotas

### PR DESCRIPTION
**Pull request by ianopolous**
Move quotas from a file to sql, and allow importing from file

### CODE REVIEW

File: src/peergos/server/space/JdbcQuotas.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        try (Connection conn = getConnection()) {
+            commands.createTable(commands.createQuotasTableCommand(), conn);
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public void setQuota(String username, long quota) {
+        try (Connection conn = getConnection();
+             PreparedStatement createuser = conn.prepareStatement(commands.insertOrIgnoreCommand("INSERT ", "INTO freequotas (name, quota) VALUES(?, ?)"));
```
Comment (kevodwyer) [inline src/peergos/server/space/JdbcQuotas.java:55]: extract sql string to constant as per other queries

File: src/peergos/server/space/JdbcQuotas.java [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            select.setString(1, username);
+            ResultSet rs = select.executeQuery();
+            return rs.next();
+        } catch (SQLException sqe) {
+            LOG.log(Level.WARNING, sqe.getMessage(), sqe);
+            throw new IllegalStateException(sqe);
+        }
+    }
+
+    public int numberOfUsers() {
```
Comment (kevodwyer) [inline src/peergos/server/space/JdbcQuotas.java:125]: select count() is the general way to do that in SQL

Comment (ianopolous) [reply src/peergos/server/space/JdbcQuotas.java:126]: done

File: src/peergos/server/space/JdbcQuotas.java (hunk 1 above)
Comment (ianopolous) [reply src/peergos/server/space/JdbcQuotas.java:56]: It can't be a static constant because it's generated dynamically.

Comment (kevodwyer) [reply src/peergos/server/space/JdbcQuotas.java:56]: I meant "INTO freequotas (name, quota) VALUES(?, ?)"

Comment (ianopolous) [reply src/peergos/server/space/JdbcQuotas.java:56]: But it doesn't make any sense on it's own

**Review by kevodwyer — APPROVED** [review]

**Pull request merged**